### PR TITLE
chore: Remove unused util functions

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

File: src/sentry/static/sentry/app/utils.jsx [hunk 1]
```diff
@@ -291,22 +223,15 @@ export {parseLinkHeader, deviceNameMapper, Collection, PendingChangeQueue, Curso
...
   objectIsEmpty,
-  trim,
   defined,
   nl2br,
   isUrl,
   escape,
   percent,
-  urlize,
   toTitleCase,
-  arrayIsEqual,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/utils.jsx]: Why keep these implementations around?

**Review by billyvg — APPROVED** [review]
Make sure to see if any of these are used in `getsentry` first

File: src/sentry/static/sentry/app/utils.jsx (hunk 1 above)
Comment (lynnagara) [reply src/sentry/static/sentry/app/utils.jsx]: it's used in this file

**Comment by lynnagara** [comment]
👍 Checked getsentry for all the removed exports

**Pull request merged**